internal/db/schema/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql

```diff
@@ -113,7 +292,11 @@ begin;
        and c.relname operator(pg_catalog.~) '^(.+_deleted)$' collate pg_catalog.default
        and pg_catalog.pg_table_is_visible(c.oid);
   $$ language sql;
+  comment on function get_deletion_tables is
+    'get_deletion_tables returns a set containing all the deleted table names by looking for '
+    'all tables that end in _deleted.';
 
+  -- Assign a "before delete" trigger on all deleted tables to run insert_deleted_id
```

Comment:
We usually use the same name for the trigger and the trigger function. Any reason for not doing that here?